src/core/mod.rs

```diff
@@ -364,9 +343,15 @@ lazy_static! {
         StacksEpoch {
             epoch_id: StacksEpochId::Epoch20,
             start_height: BITCOIN_MAINNET_FIRST_BLOCK_HEIGHT,
-            end_height: STACKS_2_0_LAST_BLOCK_TO_PROCESS + 1,
+            end_height: BITCOIN_TESTNET_STACKS_2_05_BURN_HEIGHT,
```

Comment:
Note that the prior value of end_height here would have made the `next-costs` branch broken when running on Xenon testnet: the end_height is long before the first bitcoin block height that the testnet considers